### PR TITLE
New account for tipstaff

### PR DESCRIPTION
**Pull request by SteveLinden**
Creating new account for tipstaff

### CODE REVIEW

File: policies/environments/expected.rego [hunk 1]
```diff
@@ -38,6 +38,7 @@ expected =
     "sprinkler",
     "testing",
     "tariff",
-    "xhibit-portal"
+    "xhibit-portal",
+    "tipstaff"
```
Comment (ewastempel) [inline policies/environments/expected.rego:42]: I have a feeling this needs to be added in an alphabetical order...

File: environments/tipstaff.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "name": "development",
+        "access": [
+          {
+            "github_slug": "tipstaff-team",
+            "level": "sandbox",
+            "nuke": "rebuild"
+          }
+        ]
+      },
+           {
```
Comment (ewastempel) [inline environments/tipstaff.json:14]: The indentation/formatting looks a bit funny here. I think it is correct in terms of the syntax, but it's difficult to review. 

I know we have a formatter that runs nightly now, but in the future you can format your JSON code in VSC with `Ctrl+Shift+I`. It's a good habit to have anyway.

File: policies/environments/expected.rego (hunk 1 above)
Comment (SteveLinden) [reply policies/environments/expected.rego:42]: I'll amend

**Review by ewastempel — APPROVED** [review]
LGTM

**Pull request merged**